docs/guides/expectations/advanced/how-to-create-a-new-expectation-suite-by-profiling-from-a-jsonschema-file.md

```diff
@@ -1,5 +1,100 @@
 ---
 title: How to create a new Expectation Suite by profiling from a jsonschema file
 ---
+import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
+
+The ``JsonSchemaProfiler`` helps you quickly create [Expectation Suites](../../../reference/expectations/expectations.md) from jsonschema files.
+
+<Prerequisites>
+
+  - [Set up a working deployment of Great Expectations](../../../tutorials/getting-started/intro.md)
+  - Have a valid jsonschema file that has top level object of type `object`
+
+</Prerequisites>
+
+:::warning 
+This implementation does not traverse any levels of nesting.
+:::
+
+Steps
+-----
+
+1. **Set a filename and a suite name**
+
+    ```python
```

Comment:
@cdkini Did we say we wanted to un-indent such macros as "```python", "```bash", etc. maximally all the way to the left?

Reply:
Resolving this comment per our discussion earlier today about codeblock alignment.